### ISSUE TITLE
Support for Typescript

### ISSUE THREAD

**Issue by SorenMuller** [aurelia/i18n#27]
It would be absolutely awesome, if support for Typescript were added.. :-)

**Comment by SorenMuller** [comment]
Any idea when this issue will be adressed ??

**Comment by EisenbergEffect** [comment]
In the next release....beta is coming in the next week or so. We are waiting on some dependency tooling updates which broke use...related to the Babel 6 update. Hang in there.

**Comment by nomack84** [comment]
This is a duplicate of #8
